dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScopeTest.kt

```diff
@@ -99,6 +100,9 @@ internal class RumActionScopeTest {
     @Mock
     lateinit var mockRumEventSourceProvider: RumEventSourceProvider
 
+    @BoolForgery
+    var fakeTrackFrustrations: Boolean = true
```

Comment:
```suggestion
    lateinit var fakeTrackFrustrations
```

because value will be assigned by injector